src/Reporter/JsonReporter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace FriendsOfTwig\Twigcs\Reporter;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use FriendsOfTwig\Twigcs\Validator\Violation;
+
+class JsonReporter implements ReporterInterface {
+
+    /**
+     * {@inheritdoc}
+     *
+     * @param Violation[] $violations
+     */
+    public function report(OutputInterface $output, array $violations)
+    {
+        $filename = null;
+
+        $result = new \stdClass();
+        $result->failures = count($violations);
+
+        foreach ($violations as $violation) {
+            if ($filename !== $violation->getFilename()) {
+                $filename = $violation->getFilename();
+                $entry = new \stdClass();
+                $entry->file = $filename;
+                $result->files[] = $entry;
+            }
+
+            $entry->violations[] = [
+                'line' => $violation->getLine(),
+                'column' => $violation->getColumn(),
+                'severity' => $violation->getSeverity(),
+                'type' => strtolower($violation->getSeverityAsString()),
+                'message' => $violation->getReason(),
+            ];
+        }
+
+        $output->writeln(json_encode($result, JSON_PRETTY_PRINT));
+    }
+}
```

Comment:
Add end of file please

Reply:
What do You mind with add end of file?

Reply:
If you go on `Files changes` tab, you will see on the end of file a litle logo red. On mouse over it said `Non new line at the end of file`. Just add a empty line at the end of file will fix it 😃